code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1384,6 +1395,34 @@ var/list/rank_prefix = list(\
 					break
 	return 1
 
+/mob/living/carbon/human/proc/resuscitate_notify(type)
+	if(prob(50))
```

Comment:
...may I inquire why this is here?

Reply:
This proc is called every time resusitate called so it happens every tick resuscitator is metabolized (>25 times per 5 units)
50 percent check is here to remove at least some spam, but it can't really solve whole problem.
It isn't so bad, but I can't find a better solution. Any ideas?